Intersect.Client/MonoGame/Graphics/MonoRenderer.cs

```diff
@@ -835,7 +837,7 @@ public override GameFont LoadFont(string filename)
             }
 
             // Concatenate the parts of the name except the last one to get the full name
-            name = string.Join($"{'_'}", parts.Take(parts.Length - 1));
+            name = string.Join(separator.ToString(), parts.Take(parts.Length - 1));
```

Comment:
I keep trying to stop you from casting the char into a string but you keep finding different ways to convert it to a string.

You should be using [Join\<T\>(String, IEnumerable\<T\>)](https://learn.microsoft.com/en-us/dotnet/api/system.string.join?view=net-7.0#system-string-join-1(system-string-system-collections-generic-ienumerable((-0))))

Reply:
Apparently this method doesn't actually exist in .NET Framework, so use

`string.Join('_', parts, 0, parts.Length - 1)`

Reply:
> Cannot resolve method 'Join(char, System.Collections.Generic.IEnumerable<string>)', candidates are: 
> string Join(string, params object[]) (in class String) 
> string Join(string, params string[]) (in class String) 
> string Join(string, System.Collections.Generic.IEnumerable<string>) (in class String)
> string Join<string>(string, System.Collections.Generic.IEnumerable<string>) (in class String)

Reply:
The following :

`name = string.Join("'_'", 0, parts.Length - 1);`

nor the following work

`name = string.Join('_', 0, parts.Length - 1);`

we can't use chars directly here, it has to be a char converted to it or the string right way, previously it was using a string like this:

![imagen](https://user-images.githubusercontent.com/17498701/210461188-ad2eb171-727d-4bb7-a3fa-7a60a6f7c517.png)

which takes me to the following (and works to load the fonts too)

![imagen](https://user-images.githubusercontent.com/17498701/210461263-80f153d5-32f3-4f63-bf29-381d08a483ce.png)